models/userStatus.js

```diff
@@ -223,7 +225,7 @@ const updateUserStatus = async (userId, newStatusData) => {
           newStatusData.futureStatus = {};
         }
         if (isNewStateOoo) {
-          if (newStatusData.currentStatus.from >= tommorow) {
+          if (newStatusData.currentStatus.from >= tomorrow) {
```

Comment:
Can you please explain why this check is needed? Why are we checking the currentStatus's start time with the next day's date?